styles.css

```diff
@@ -52,4 +52,6 @@
 /* Custom styles */
 .flex-container {
   display: flex;
+	/* min-height: 100vh; */
```

Comment:
Could you please add some additional information why this property should be removed? Also, It would be better to remove it at all instead of commenting.